Applications/ClientControls.Net4/Common/Client/SubscribeEventsDlg.cs

```diff
@@ -659,12 +659,12 @@ private void OnPublishStatusChanged(object sender, EventArgs e)
 
             try
             {
-                if (m_subscription.PublishingStopped)
+                if (e.Status == PublishStateChangedMask.Stopped)
```

Comment:
the mask is designed to be used as a bitmask, but here it is checked for equality (not whether a bit is set). The where use list shows that this seems to be ok, but what if, at a later point in time, someone ORs the Recovered state with something else? If that cannot happen that's ok for me, but this way of using a mask looks at least suspicious to me.
Also there is no else branch. Again, when that's ok I'm fine with it.